### PR TITLE
Initial commit of the paladin quest walkthrough.

### PR DESCRIPTION
**Pull request by jayRyan24**

### CODE REVIEW

File: paladin-quests.lic [hunk 1]
```diff
@@ -0,0 +1,105 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#cleric-quests
```
Comment (rpherbig) [inline paladin-quests.lic]: Should this be `...#paladin-quests`?

Comment (jayRyan24) [reply paladin-quests.lic]: 📦

File: paladin-quests.lic [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    elsif args.ease
+      ease
+    elsif args.renewal
+      renewal
+    else
+      echo '***UNKNOWN QUEST NAME***'
+    end
+  end
+
+  def warding
```
Comment (rpherbig) [inline paladin-quests.lic]: According to https://elanthipedia.play.net/Glyph_of_Warding_walkthrough this requires a Pristine soul state. Do you want to try to check for that?

File: paladin-quests.lic [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    walk_to(8228)
```
Comment (rpherbig) [inline paladin-quests.lic]: Does the guildleader give a visa for f2p paladins?

Comment (Sarvatt) [reply paladin-quests.lic]: yeah for Glyph of Warding, Glyph of Light, and Glyph of Mana

File: paladin-quests.lic (hunk 2 above)
Comment (jayRyan24) [reply paladin-quests.lic]: Nah, I say let them figure out how to do it and then do it. We also don't account for money or anything like that, or them being the proper level. They need to read requirements, meet them, and then do it.

File: paladin-quests.lic (hunk 3 above)
Comment (jayRyan24) [reply paladin-quests.lic]: He does, they'll need it before doing this. I won't be holding hands *that* much. :D

**Pull request merged**